composer.json

```diff
@@ -23,18 +26,61 @@
       "email": "wolfgang@woda.at"
     }
   ],
-  "type": "wordpress-plugin",
   "support": {
     "issues": "https://github.com/wwwoda/woda-scripts-styles-loader/issues",
     "source": "https://github.com/wwwoda/woda-scripts-styles-loader"
   },
-  "require": {
-    "php": ">=7.2.0",
-    "ext-json": "*"
-  },
   "autoload": {
     "psr-4": {
-      "Woda\\WordPress\\ScriptsStylesLoader\\": "includes/"
+      "Woda\\WordPress\\ScriptsStylesLoader\\": "includes"
     }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "Woda\\Test\\WordPress\\ScriptsStylesLoader\\": "test"
+    }
+  },
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org"
+    }
+  ],
+  "require": {
+    "php": ">=7.2"
```

Comment:
in `includes/HashFile.php` werden `json_*` funktionen verwenden,  also wäre auch `ext-json` nötig